user/api.go

```diff
@@ -293,6 +295,8 @@ func (a *Api) SetHandlers(prefix string, rtr *mux.Router) {
 
 	rtr.Handle("/token/{token}", varsHandler(a.ServerCheckToken)).Methods("GET")
 
+	rtr.Handle("/sso/{service}", varsHandler(a.Get3rdPartyToken)).Methods("POST")
```

Comment:
actually I have a doubt on the name here, as this is not true SSO. Shall we name it differently? ext_token?